ubuntu-advantage

```diff
@@ -80,6 +80,8 @@ Commands:
  enable-livepatch <TOKEN>          enable the Livepatch service
  disable-livepatch [-r]            disable the Livepatch service. With "-r", the
                                    canonical-livepatch snap will also be removed
+ install-cc <TOKEN>                enable the commoncriteria PPA repository and
+                                   install the ubuntu-commoncriteria .DEB package
```

Comment:
Please add a common-criteria one-line description under "Currently available are" in the usage() function, after livepatch.